content/id/docs/concepts/cluster-administration/monitoring.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Metrik-Metrik untuk Control-Plane Kubernetes 
+content_template: templates/concept
+weight: 60
+aliases:
+- controller-metrics.md
```

Comment:
For the localization it's OK to omit this.

Reply:
Ok!